examples/git-data/zarf.yaml

```diff
@@ -9,7 +9,7 @@ components:
     required: true
     repos:
       # The following performs a full Git Repo Mirror with `go-git` (internal to Zarf)
-      - https://github.com/defenseunicorns/zarf-public-test.git
```

Comment:
The tests will need to be updated to properly support this.

Reply:
sounds good!